resources/views/modals/submit_patrol_log_modal.blade.php

```diff
@@ -45,48 +45,57 @@
                   <div class="input-group-addon input-group-append"><i class="mdi mdi-clock input-group-text"></i></div>
                 </div>
               </div>
               <label id="patrol-end_time-error" class="error mt-2 text-danger" for="patrol-end-time" hidden></label>
             </div>
             <div class="form-group">
-              <label>Patrol Description (required)</label>
-              <textarea class="form-control" id="patrol-details" rows="6"></textarea>
+              <label>Patrol Description<sup class="text-danger">*</sup></label>
+              <textarea class="form-control" id="patrol-details" rows="6" required></textarea>
               <label id="patrol-details-error" class="error mt-2 text-danger" for="patrol-details" hidden></label>
             </div>
               <div class="form-group">
-                  <label>How many priorities did you do? (required)</label>
+                  <label>How many priorities did you do?<sup class="text-danger">*</sup></label>
                   <div class="input-group" id="patrol-priorities2" data-target-input="nearest">
-                      <input type="text" class="form-control" id="patrol-priorities"/>
+                      <input type="number" class="form-control" id="patrol-priorities" required>
                       <div class="input-group-addon input-group-append"><i class=" mdi mdi-apple-keyboard-caps input-group-text"></i></div>
                   </div>
                   <label id="patrol-priorities-error" class="error mt-2 text-danger" for="patrol-priorities" hidden></label>
               </div>
               <div class="form-group">
-                  <label>Select Patrol Area (required)</label>
-                  <select class="js-example-basic-multiple" multiple="multiple" id="patrol-area" style="width:100%">
+                  <label>Select Patrol Area<sup class="text-danger">*</sup></label>
+                  <select class="js-example-basic-multiple" multiple="multiple" id="patrol-area" style="width:100%" required>
                       @foreach($constants['patrol_area'] as $patrol_area => $value)
                           <option value="{{$value}}">{{$value}}</option>
                       @endforeach
                   </select>
                   <label id="patrol-area-error" class="error mt-2 text-danger" for="patrol-area" hidden></label>
               </div>
+              <div class="form-group">
+                  <label>Flag this Patrol Log?</label>
+                  <div class="form-check form-check-info">
+                      <label class="form-check-label"><input type="checkbox" id="flag-patrol-log"> Flag Patrol Log <i class="input-helper"></i><i class="input-helper"></i> <button class="btn btn-outline-secondary" style="padding: 2px;" data-toggle="tooltip" data-placement="bottom" title="By flagging your Patrol Log, you are indicating to staff that you would like your Patrol Log reviewed." disabled>?</button></label>
```

Comment:
tooltip doesn't really work, check why,
you can find the documentation for all elements here:
https://www.bootstrapdash.com/demo/corona/jquery/template/modern-vertical/index.html

Reply:
I disagree